src/components/ebay-select/index.js

```diff
@@ -120,11 +120,10 @@ function setSelectedOption(optionValue) {
         } else {
             option.selected = false;
         }
-        this.update();
         return option;
     });
-
     this.setState('options', options);
+    this.update();
```

Comment:
@seangates This change is not related to the button component. Is it still fine?